jujugui/static/gui/src/app/state/test-state.js

```diff
@@ -19,6 +19,247 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 'use strict';
 
 describe('State', () => {
+
+  const USER_STATE_URLS = [{
```

Comment:
Thanks for this comment, what do you think about renaming this to `const userStateTests` as they are actually "tests" in a table test scenario?